netty/src/main/java/io/grpc/transport/netty/Utils.java

```diff
@@ -72,6 +70,9 @@
       HttpUtil.CONTENT_TYPE_GRPC.getBytes(UTF_8));
   public static final ByteString TE_HEADER = new ByteString(HttpUtil.TE.name().getBytes(UTF_8));
   public static final ByteString TE_TRAILERS = new ByteString(HttpUtil.TE_TRAILERS.getBytes(UTF_8));
+  public static final ByteString USER_AGENT = new ByteString("user-agent".getBytes(UTF_8));
+  public static final ByteString DEFAULT_USER_AGENT = new ByteString(
+          "grpc-java-netty".getBytes(UTF_8));
```

Comment:
Have you thought about adding version? We could add the version to the JAR Manifest as Implementation-Version of io/grpc and then access it here with SomeGrpcClass.class.getPackage().getImplementationVersion()